internal/github_client/types.go

```diff
@@ -1,10 +1,14 @@
 package github_client
 
-import "github.com/google/go-github/github"
+import (
+	"github.com/google/go-github/github"
```

Comment:
nit: This uses a very old version of the library. Perhaps this is a good time to also upgrade that 😄

Reply:
Done!